src/VirtoCommerce.Platform.Web/Security/ServiceCollectionExtensions.cs

```diff
@@ -47,5 +51,22 @@ public static IServiceCollection AddSecurityServices(this IServiceCollection ser
 
             return services;
         }
+
+        public static void AddPlatformDataProtection(this IServiceCollection services, IConfiguration config)
+        {
+            var redisConnectionString = config.GetConnectionString("RedisConnectionString");
+            if (!string.IsNullOrEmpty(redisConnectionString))
+            {
+                var redis = ConnectionMultiplexer.Connect(redisConnectionString);
```

Comment:
try to avoid repeating  call  `ConnectionMultiplexer.Connect(redisConnectionString);` because the redis connection is established before by this line https://github.com/VirtoCommerce/vc-platform/pull/2152/files#diff-0b09b0eb927ab079e191d396c4b505ec9edf25b0b6576f03ce303973426bee3eR85